pkg/preflight/assets/host-preflights.yaml

```diff
@@ -394,3 +394,11 @@ spec:
           - fail:
               when: "ubuntu = 22.04"
               message: "Collectd addon not supported on ubuntu 22.04"
+    - hostOS:
+        checkName: "Docker Support"
+        exclude: '{{kurl and .Installer.Spec.Docker.Version (semverCompare "< 20.10.17" .Installer.Spec.Docker.Version) | not }}'
```

Comment:
can you flip this so its as you would read it

"exclude if (docker version does not exist) or (docker version >= 20.10.17)"

Reply:
Is this better? if () or () seems complicated.